File: Arrays & Hashing/Valid Anagram  .py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        dic = {}
        dic1 ={}
        for c in s : 
            dic[c] = dic.get(c, 0) + 1
        for c in t : 
            dic1[c] = dic1.get(c, 0) + 1
        if dic==dic1:
            return True
        else:
            return False

File: Arrays & Hashing/c++/Valid Anagram  .cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        map<char,int >dic1;
        map<char,int >dic2;
        for (int i=0;i<size(s);i++)
        {
            if (s[i])
            dic1[s[i]]=dic1+1
        }
        for (int i=0;i<size(t);i++)
        {
            dic2[s[i]]=dic2+1
        }
        if (dic1 == dic2 )
        {
            return true
        }
        else
            return false
        
    }
};